### ISSUE TITLE
Idea/ proposal form: Keyboard users can't access CK editor formatting options

### ISSUE THREAD

**Issue by sblindow** [liqd/a4-meinberlin#6288]
**URL:** 

Idea/ Proposal form

**user:** 

Authenticated keyboard users

**expected behaviour:** 

Users should be able to use the CK editor's formatting options without a mouse via keyboard navigation.

**behaviour:** 

The tab order skips over the text formatting buttons in the CK editor, making these features inaccessible to keyboard-only users.

**Comment by vellip** [comment]
Accessibility is built into CK editor, there is a way to reach these buttons pressing alt + f10, and further details are announced to screenreaders as described here https://ckeditor.com/docs/ckeditor4/latest/guide/dev_a11y.html

**Comment by CarolingerSeilchenspringer** [comment]
@vellip thanks for clarifying. @sblindow maybe you want to take over this link into the testing manual as a reference on how to test CKeditor?
